CHANGELOG.md

```diff
@@ -21,10 +21,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-api-python3#89](https://github.com/cyberark/conjur-api-python3/issues/89)
 
 ### Changed
+- The CLI and SDK now uses a system's native credential store to save credentials instead of a netrc file. If a store
+  is not available, the credentials will be saved to the netrc as a fallback.
```

Comment:
@moticless would like your 👀

Reply:
Maybe add a link to supported keyring backends: https://pypi.org/project/keyring/